src/Caching/StackExchangeRedis/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+
+  <PropertyGroup>
+    <!-- These projects depend on a 3rd party source -->
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
```

Comment:
nit: Again, why is this setting here❔ We have too many `Directory.Build.*` files and they make settings harder to understand.